tests.py

```diff
@@ -13,18 +13,16 @@ def test_un():
         r'1	a',
         r'2	\t',
         r'3	\n',
-        r'4	\\',
+        r'4	\\\\',
```

Comment:
With `\\` in a raw string, this actually tested the right thing: that a final pair of backslashes was okay but a dangling final backslash was not. I am not sure that `\\\\` tests anything different but it does seem less clear.